Backend/ManagerLayer/Logout/LogoutManager.cs

```diff
@@ -0,0 +1,82 @@
+﻿using System;
+using System.Text;
+using System.Threading.Tasks;
+using System.Net.Http;
+using DataAccessLayer.Models;
+using System.Security.Cryptography;
+using DataAccessLayer.Database;
+using ServiceLayer.Services;
+using Newtonsoft.Json;
+
+namespace ManagerLayer.Logout
+{
+    public class UserLogoutPayload
+    {
+        // Included in signature
+        public Guid ssoUserId { get; set; }
+        public string email { get; set; }
+        public long timestamp { get; set; }
+
+        // Excluded from signature
+        public string signature { get; set; }
+
+        // Generate string to be signed
+        public string PreSignatureString()
+        {
+            string acc = "";
+            acc += "ssoUserId=" + ssoUserId + ";";
+            acc += "email=" + email + ";";
+            acc += "timestamp=" + timestamp + ";";
+            return acc;
+        }
+    }
```

Comment:
This class can be removed now. We're using SignatureService.

Reply:
I have removed them.